dashboard/app/views/levels/_level_group.haml

```diff
@@ -1,13 +1,11 @@
 - app = 'level_group'
 - data = @level.properties
 
-:javascript
-  #{render template: 'levels/multi.js', formats: [:js]}
-  #{render template: 'levels/textMatch.js', formats: [:js]}
-  #{render template: 'levels/levelGroup.js', formats: [:js]}
+- content_for(:head) do
+  =javascript_include_tag 'js/assessments'
 
+:javascript
   window.levelData = #{data.to_json};
-  window.levelGroup = {levels: {}};
```

Comment:
Personal preference: I'd love to see this as an explicit call to an initialization function, something like `window.dashboard.loadLevelGroupData(#{data.to_json})`.  Okay either way though.